code/modules/events/sayu_artifactdust.dm

```diff
@@ -0,0 +1,23 @@
+// ARTIFACT METEORS -- Collen
+
+/datum/round_event_control/meteor_wave/artifact
+	name = "Artifact Meteor"
+	typepath = /datum/round_event/meteor_wave/artifact
+	weight = 50 // This value needs some tweaking, not really sure what to put it at.
```

Comment:
That is still too much for an event like this. The weight of "some lights overload" is 40. Put it between 5 to 10.